src/cosalib/gcp.py

```diff
@@ -26,6 +26,7 @@ def remove_gcp_image(gcp_id, json_key, project):
             '--json-key', json_key,
             '--project', project
         ])
+        print(f"GCP: successfully removed image {gcp_id}")
     except SystemExit:
         raise Exception("Failed to remove image")
```

Comment:
I guess for consistency with the AWS one let's add the `gcp_id` to this exception.